### ISSUE TITLE
Skip error about non-existing file if deleted before instant upload

### ISSUE THREAD

**Issue by PVince81** [nextcloud/android#7810]
### Steps to reproduce
1. Setup instant upload
2. Go offline (or off wifi if you set it up for "sync only on wifi")
3. Take a shitty picture
4. Wait for NC app to detect it and skip it
5. Using file manager, go and delete that picture, because it wasn't that good anyway
6. Go back online by enabling wifi
7. Wait for sync
8. Check the "Uploads" section

### Expected behaviour
No errors in uploads section.

### Actual behaviour
An error appears that the deleted file cannot be found.

### Version
Nextcloud app 3.14.1
Android 10
Nokia 6.1

### Notes

I'm guessing that the app is keeping track of pending files to upload after a picture has been taken, and then goes through that list when going back online. That code would need to be adjusted to not trigger/display an error in the log for files that got deleted (but only do that for the instant upload workflow, not for the regular syncing logic)

**Comment by tobiasKaminsky** [comment]
Is this really what we want? To hide this error?

**Comment by PVince81** [comment]
> Is this really what we want? To hide this error?

Yes. Because as a user I already purposefully delete the picture while offline because I didn't like it.
So no point in reminding me that the picture isn't there when going online as it will make me believe (or scare me) that something is wrong with my data.
